src/components/clusters/ClusterDetailsMultiRegion/components/ClusterDetailsTop/components/RecommendedOperatorsAlert/ProductCardView.tsx

```diff
@@ -24,8 +26,13 @@ const ProductCardView = ({
   title,
   selectedCardTitle,
   learnMoreLink,
+  isHidden = false,
 }: ProductCardViewProps) => (
-  <div className={`${title ? `${title}-` : ''}product-card-view`}>
+  <div
+    className={`${title ? `${title}-` : ''}product-card-view`}
+    data-testId={`${title ? `${title}-` : ''}product-card-view`}
```

Comment:
Couldn't think of any other way to select this div other then using a data-testId, in order to verify that the 'hidden' attribute has been added to the element. Open to hearing other opinions 😄

Reply:
Sounds good. But why do we have a className too?

Reply:
I don't think it is needed 
Removed 👍🏼